packages/web-components/tests/e2e-storybook/cypress/integration/masthead/masthead-l1.e2e.js

```diff
@@ -183,4 +184,38 @@ describe('cds-masthead | with L1 (desktop)', () => {
       })
   });
 
+  it('should have horizontal scroll for L1 items working', () => {
+    cy.viewport(1100, 780);
+    
+    cy.get(_selectors.l1)
+      .shadow()
+      .find(_selectors.l1ScrollNextArrow)
+      .click()
+      .get(_selectors.l1)
```

Comment:
Technically it's not safe to chain commands after `click()` (see [docs](https://docs.cypress.io/api/commands/click)).
```suggestion
      .click();
    cy.get(_selectors.l1)
```